include/event_manager.h

```diff
@@ -30,6 +30,26 @@
 extern "C" {
 #endif
 
+/**
+ * @brief List of bits in event type flags.
+ */
+enum event_type_flags {
+   EVENT_FLAGS_USER_SETTABLE_START,
+   EVENT_FLAGS_INIT_LOG_ENABLE = EVENT_FLAGS_USER_SETTABLE_START,
+   EVENT_FLAGS_TRACEABLE,
+   EVENT_FLAGS_SYSTEM_START, /**< Used to assert the user trying to set flags that should not be changed by the user */
```

Comment:
Comment is not valid

Reply:
But it should be used, if we get rid of the flags definition as a variable.